### PR TITLE
Exercise cache when deleting muscles

### PR DESCRIPTION
**Pull request by betsybeth**
##### Delete a muscle it will reflect in exercises that it was deleted

### CODE REVIEW

**Review by Kalela — APPROVED** [review]
LGTM

File: wger/exercises/signals.py [hunk 1]
```diff
@@ -36,6 +36,14 @@ def delete_exercise_image_on_delete(sender, instance, **kwargs):
     instance.image.delete(save=False)
 
 
+@receiver(post_delete, sender=Muscle)
```
Comment (frankip) [inline wger/exercises/signals.py:39]: LGTM

**Review by frankip — APPROVED** [review]

**Review by FelixWambiri — APPROVED** [review]
LGTM

**Review by meshnesh — APPROVED** [review]
good job completing the task

**Review by mutoromike — APPROVED** [review]
LGTM

**Pull request merged**